src/index.js

```diff
@@ -5,7 +5,16 @@ if(new URLSearchParams(window.location.search).has("p")) {
   import("./Video.js")
     .then(({Initialize}) =>
       Initialize({target: document.getElementById("app")})
-    );
+    ).catch(error => {
```

Comment:
instead of just catching the error out here, catch it within the video component and render an error page. This is a more general need for error handling in this app, unrelated to this specific feature, but might as well do it